pkg/minikube/cni/flannel.go

```diff
@@ -51,12 +50,6 @@ func (c Flannel) String() string {
 
 // Apply enables the CNI
 func (c Flannel) Apply(r Runner) error {
-	// Mostly applicable to the 'none' driver
-	_, err := r.RunCmd(exec.Command("stat", "/opt/cni/bin/portmap"))
-	if err != nil {
-		return errors.Wrap(err, "required 'portmap' CNI plug-in not found")
```

Comment:
Is `portmap` no longer required for flannel?

Reply:
It is still required, as part of the CNI config

Reply:
ok, the error may occur, keep these lines.